scene/3d/mesh_instance_3d.cpp

```diff
@@ -34,6 +34,11 @@
 #include "core/core_string_names.h"
 #include "physics_body_3d.h"
 
+#include "core/object/undo_redo.h"
+#ifdef TOOLS_ENABLED
+#include "editor/plugins/node_3d_editor_plugin.h"
+#endif
```

Comment:
Including editor files in `scene/` code is not allowed (#29730). You should come up with a way that doesn't involve this.

Although looking at the code, it might be tricky to handle as the editor isn't involved in the method aside from calling it.